rancher/charts
# [dev-v2.7] Update node-feature-discovery to v0.13.2

## Issue: 
https://github.com/rancher/rke2/issues/4153


## Problem

## Solution


## Testing


## Engineering Testing
### Manual Testing


### Automated Testing


## QA Testing Considerations
node-feature-discovery is not installed directly but only as a dependency of the sriov chart.
The installation procedure is described here: 
https://confluence.suse.com/display/RKE2K3S/CNI+Test+Plan?preview=/1120305984/1120305985/how_to_run_sriov(1).txt

For the purpose of testing only nfd, it is enough to install on one machine and check that the node gets labeled by node-feature-discovery.

The labels added by node-feature-discovery all follow this format:
```
feature.node.kubernetes.io/<feature> = <value>
```
### Regressions Considerations


## Backporting considerations
